forge-gui/res/cardsfolder/upcoming/splitskin_doll.txt

```diff
@@ -0,0 +1,8 @@
+Name:Splitskin Doll
+ManaCost:1 W
+Types:Artifact Creature Toy
+PT:2/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters, draw a card. Then discard a card unless you control another creature with power 2 or less.
+SVar:TrigDraw:DB$ Draw | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | NumCards$ 1 | Mode$ TgtChoose | IsPresent$ Creature.Other+YouCtrl+powerLE2 | PresentCompare$ EQ0

```

Comment:
bad condition

Reply:
Honestly, is there a reason for having so many slightly different syntaxes?

Reply:
you could search if there's a card that first has a restriction (IsPresent etc.) to cast/activate and then a resolving condition on the same line (ConditionCheckSVar, ...)
that would be an answer